danger-kotlin-library/src/main/kotlin/systems/danger/kotlin/models/gitlab/GitLab.kt

```diff
@@ -38,7 +38,7 @@ data class GitLabMergeRequest(
     @SerialName("allow_maintainer_to_push")
     val allowMaintainerToPush: Boolean = false,
     @SerialName("approvals_before_merge")
-    val approvalsBeforeMerge: Int = 0,
+    val approvalsBeforeMerge: Int? = 0,
```

Comment:
Can you please also update the CHANGELOG?

Reply:
Done

Reply:
It fails on `e: Unable to compile C bridges`